spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -9,7 +9,7 @@ import {
   AnalyticsContext,
   FailedCaptureAttemptsContextProvider,
 } from '@18f/identity-document-capture';
-import { createEvent, waitFor } from '@testing-library/dom';
+import { createEvent, waitFor, screen } from '@testing-library/dom';
```

Comment:
[LWIL](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen) the creator of React Testing Library advocates for using `screen`.